collect_app/src/androidTest/java/org/odk/collect/android/test/FormLoadingUtils.java

```diff
@@ -110,4 +97,31 @@ protected void afterActivityLaunched() {
             }
         };
     }
-}
+
+    private static void copyForm(String formFilename, String formAssetPath) throws IOException {
+        String pathname = Collect.FORMS_PATH + formFilename;
```

Comment:
This should probably have a `File.separator` in there right?

Reply:
When I extracted the code in that method from the various places it'd been copied to, I did expect that and added it but it didn't work? I didn't dig any deeper, though. But yes, it is curious that all paths are currently `/sdcard/odk/forms<formname>/` rather than `/sdcard/odk/forms/<formname>/`.